libs/gi-localization/assets/locales/en/char_Furina.json

```diff
@@ -0,0 +1,12 @@
+{
+  "fanfarePoints": "Fanfare",
+  "memberMult_": "Salon Members Mult.",
+  "numOffer": "Number of characters who offered HP",
+  "c1Inc": "Max Fanfare Increase",
```

Comment:
What determines if we titlecase a string vs just regular capitalization?

Reply:
Datamined text we use for nodes `text` for nodes is often in title case, so custom ones need to be formatted like that too for consistency.

With conditionals, I realized it's basically just plain text and might as well be formatted as such.